app/serializers/review_serializer.rb

```diff
@@ -0,0 +1,14 @@
+class ReviewSerializer < BaseSerializer
+  type :review
+
+ 	attributes(
+		:id,
+		:user_id,
+		:item_id,
+    :body,
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/ArgumentAlignment: Align the arguments of a method call if they span more than one line.